Pipfile

```diff
@@ -0,0 +1,21 @@
+[[source]]
+url = "https://pypi.python.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+PyYAML = "~=5.3.1"
+gevent = "~=20.9.0"
+gunicorn = "~=19.9.0"
+Flask = "~=1.1.1"
+Pillow = "~=7.2.0"
+logging-utilities = "~=1.0.0"
```

Comment:
I would use the latest version here: `1.2.1`

Reply:
:+1: probably also needs to be changed for `service-qrcode` and `service-shortlink`

Reply:
Not needed, the new features are not required, but has here it is a new Pipfile I would suggest to take the latest as a good practice ;-)